flasher_stub/ld/rom_32s3.ld

```diff
@@ -529,6 +529,8 @@ ets_isr_unmask = 0x40001b90;
  ***************************************/
 
 /* Functions */
+
+memcpy = 0x40056f44;
```

Comment:
This looks like an address past the ROM pointer table, it's probably the real address of `memcpy` rather than the table address. It may change in a future ROM revision. It's more reliable to use the address from the ROM pointer table:
```suggestion
memcpy = 0x400011f4;
```